app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonFragment.kt

```diff
@@ -0,0 +1,91 @@
+package org.oppia.android.app.resumelesson
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.android.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** Fragment that allows the user to resume a saved exploration. */
+class ResumeLessonFragment :
+  InjectableFragment() {
```

Comment:
Should be on one line (in general, prefer to keep all statements on one line unless they need to be wrapped.

Ditto elsewhere in the PR (including for ResumeLessonActivity).

Reply:
done